ColqueOlimpia/Outfitaplication/src/GeneralClasses/Boots.java

```diff
@@ -0,0 +1,9 @@
+package GeneralClasses;
+
+public class Boots extends Footwear {
+    String typeBoots;
+    public Boots(String color, String size, String brand, boolean isFormal, String fabrique, String typeBoots ) {
```

Comment:
You can add an empty constructor to your classes, so you can use setters and getters properly. Also don't forget to define them